test/e2e/e2e_test.go

```diff
@@ -91,6 +91,7 @@ func init() {
 	flag.BoolVar(&testContext.GatherKubeSystemResourceUsageData, "gather-resource-usage", false, "If set to true framework will be monitoring resource usage of system add-ons in (some) e2e tests.")
 	flag.BoolVar(&testContext.GatherLogsSizes, "gather-logs-sizes", false, "If set to true framework will be monitoring logs sizes on all machines running e2e tests.")
 	flag.BoolVar(&testContext.GatherMetricsAfterTest, "gather-metrics-at-teardown", false, "If set to true framwork will gather metrics from all components after each test.")
+	flag.StringVar(&testContext.OutputPrintType, "output-print-type", "hr", "Comma separated list: 'hr' for human readable summaries 'json' for JSON ones.")
```

Comment:
I don't think there's a harm in allowing to pass a list, but I can make it exclusive if you want. The use case I had in mind is that for Jenkins runs we probably want to have both (similarily to what we have to top latency metrics and e2e startup latnecy), but when running localy json is probably unneeded.